zeppelin/zeppelin.sh

```diff
@@ -19,9 +19,15 @@
 
 set -euxo pipefail
 
-readonly NOT_SUPPORTED_MESSAGE="Zeppelin initialization action is not supported on Dataproc ${DATAPROC_VERSION}.
+# Detect dataproc image version from its various names
+if (! test -v DATAPROC_IMAGE_VERSION) && test -v DATAPROC_VERSION; then
+  DATAPROC_IMAGE_VERSION="${DATAPROC_IMAGE_VERSION}"
+fi
+
+
+readonly NOT_SUPPORTED_MESSAGE="Zeppelin initialization action is not supported on Dataproc ${DATAPROC_IMAGE_VERSION}.
```

Comment:
Missing the trailing ", and you might want to it actionable by adding "use Zeppelin optional component please".

Reply:
no, the string is continued to the next line.  I thought the same thing when I looked at it, though, so you're in good company.  Erick corrected me.  The full text reads:

readonly NOT_SUPPORTED_MESSAGE="Zeppelin initialization action is not supported on Dataproc ${DATAPROC_IMAGE_VERSION}.
Use Zeppelin Component instead: https://cloud.google.com/dataproc/docs/concepts/components/zeppelin"

Would you like it corrected from `Use Zeppelin Component instead` to `Use Zeppelin Optional Component instead` ?

Reply:
Added Use Zeppelin Optional Component instead